zerver/webhooks/reviewboard/view.py

```diff
@@ -0,0 +1,176 @@
+from typing import Any, Dict, Iterable, Optional
+
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.webhooks.common import check_send_webhook_message, \
+    validate_extract_webhook_http_header, UnexpectedWebhookEventType
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_error, json_success
+from zerver.lib.validator import check_dict, check_string
+from zerver.models import UserProfile
+
+REVIEW_REQUEST_PUBLISHED = """
+New review request [#{id}: {review_request_title}]({review_request_url}) published by **{user_name}**:
+"""
+
+REVIEW_REQUEST_REOPENED = """
+Review request [#{id}: {review_request_title}]({review_request_url}) reopened by **{user_name}**:
+"""
+
+REVIEW_REQUEST_CLOSED = """
+Review request [#{id}: {review_request_title}]({review_request_url}) closed by **{user_name}**:
+"""
+
+REVIEW_PUBLISHED = ('[New review]({review_url}) on review request [#{id}: {review_request_title}]'
+                    '({review_request_url}) published by **{user_name}**:\n\n'
+                    '**Review**:\n'
+                    '``` quote\n'
+                    '{review_body_top}\n'
+                    '```')
```

Comment:
I think this should be worded more like "[Eeshan Garg] requested review on [#{id}: {review_request_title}](url)"; the phrase "published by" feels awkward.

I think the same action verb "person closed review [stuff]()" approach for closed/reopened/replied makes sense as well.

Reply:
The reason I used this wording was that I was following Slack's format of ReviewBoard notifications. Anyway, I agree that your suggestion makes the messages less awkward/simpler! Thanks! :)